content/en/docs/reference/command-line-tools-reference/feature-gates/separate-cache-watch-rpc.md

```diff
@@ -0,0 +1,14 @@
+---
+title: SeparateCacheWatchRPC
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
+---
+Allow watch cache to create a watch on a dedicated RPC.
+This prevents watch cache from being starved by other watches.
```

Comment:
As of today, this feature gate is indeed for the watch cache inside the kube-apiserver. (can it change in the future ?)

When the FG is enabled, the watch cache establishes a separate gRPC stream (for watch and progress notification requests) to the database, in this case, etcd. 

The gRPC stream is a bidirectional (reading and writing data) connection between the client (in this case, the watch cache) and the server (in this case, etcd). A separate connection dedicated solely to the watch cache prevents starvation of that connection because (other) watch requests that bypass the watch cache (e.g., those going directly to etcd) do not use the same gRPC stream/connection. When this FG is turned off watch requests to etcd from different clients (say kubectl and watch cache) could use the same underlying gRPC stream/connection.

I hope this somewhat clarifies the usage of this FG and that I have also understood it correctly :)